syntax = "proto3";
package tinkoff.proto;

// Request
message CompleteDealStepRequest {
  string deal_id = 1;
  string step_id = 2;
}
